src/components/sidebar/InputContainer.tsx

```diff
@@ -73,19 +73,33 @@ class InputContainer extends React.Component<IInputProps, IInputState> {
         this.updateValue(value, validatorResult);
     }
 
-    private renderDisabledContent = (type: inputType) => {
-        return (
-            <div>
-                {
-                    type === 'date' ?
-                        moment(this.props.value!).format('DD.MM.YYYY') :
-                        this.props.value!
-                }
-            </div>
-        );
+    private renderDisabledContent = () => {
+        const type = this.props.type || 'text';
+
+        if (type === 'text' || type === 'number') {
+            return (
+                <div className={s.editingDisabled}>
+                    {this.props.value!}
+                </div>
+            );
+        }
+        if (type === 'date') {
+            return (
+                <div>
+                    {
+                        type === 'date' ?
+                            moment(this.props.value!).format('DD.MM.YYYY') :
+                            this.props.value!
+                    }
+                </div>
+            );
+        }
+        throw new Error(`inputContainer type not supported: ${type}`);
```

Comment:
Type is typed as text/number/date but if we get new types, they might not be covered here so that is why I throw error (sanity check). Added comment: "Should not occur"

Other option would be to return null but then it would hide an unwanted state